### PR TITLE
Swapping fonts

### LINKED ISSUE #654: Replace Vollkorn with Merriweather
The team elected to change typefaces: Vollkorn -> Merriweather

### PR DESCRIPTION
**Pull request by burgwyn**
This closes #654

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,8 +25,8 @@
     "redux-saga": "^0.15.4",
     "redux-thunk": "^2.2.0",
     "shortid": "^2.2.8",
+    "typeface-merriweather": "^0.0.35",
```
Comment (mjoyce91) [inline package.json]: Does `yarn` need to be updated?

Comment (burgwyn) [reply package.json]: The lockfile or the larger dependency that is now v `1.1.0`?

Comment (mjoyce91) [reply package.json]: lockfile

Comment (burgwyn) [reply package.json]: Yes, it gives us consistency across machines and [other awesome benefits](https://yarnpkg.com/lang/en/docs/yarn-lock/)!

Comment (mjoyce91) [reply package.json]: Oops, didn't see that `yarn.lock` was in fact updated!

**Pull request merged**